app/pods/components/certificate-comp/template.hbs

```diff
@@ -74,4 +74,10 @@
     @close={{action (mut showShareModal) false}}
   />
 </ModalDialog>
-{{/if}}
\ No newline at end of file
+{{/if}}
+
+<Blockers::Certificate 
+  @runAttempt={{runAttempt}} 
+  @checkAndGenerate={{checkAndGenerate}} 
```

Comment:
change the argument name of the blocker to `show` , also pass in `generateCertificateTask` as an argument (`onContinue`)



```
<Blockers::Certificate
   @runAttempt={{runAttempt}} 
   @show={{showCertificateGenerationBlocker}}
   @close={{action (mut showCertificateGenerationBlocker) false}}
   @onContinue={{perform generateCertificateTask}}
/>
```

Reasons:
- Avoid two-way data binding. Pass data one-way (from parent to child) if child needs to mutate data it calls a function which is also passed in by parent. (Data Down Actions Up)
- Treat props (or component arguments) as immutatable
- better naming